### ISSUE TITLE
Table styles don't work in online word

### ISSUE THREAD

**Issue by tripodsan** [dolanmiu/docx#877]
Online word has problems with custom table styles.
for example the https://github.com/dolanmiu/docx/blob/master/demo/25-table-xml-styles.ts demo produces this in word:

![image](https://user-images.githubusercontent.com/917628/113470087-c1f09080-948d-11eb-978b-687dd2f72d6b.png)

but this in online word:

![image](https://user-images.githubusercontent.com/917628/113470094-cddc5280-948d-11eb-9906-3475cd1e2db3.png)

---
tested with `docx-6.0.3`

**Comment by tripodsan** [comment]
I think the problem is the column widths:

```json
{
  "deleted": false,
  "rootKey": "w:tblGrid",
  "root": [
    {
      "deleted": false,
      "rootKey": "w:gridCol",
      "root": [
        {
          "deleted": false,
          "rootKey": "_attr",
          "root": {
            "w": 100
          },
          "xmlKeys": {
            "w": "w:w"
          }
        }
      ]
    },
    {
      "deleted": false,
      "rootKey": "w:gridCol",
      "root": [
        {
          "deleted": false,
          "rootKey": "_attr",
          "root": {
            "w": 100
          },
          "xmlKeys": {
            "w": "w:w"
          }
        }
      ]
    }
  ]
}
```

if you don't specify them, they are initialized with `100`, which is very small :-) as they are `1/1440` of an inch:
http://officeopenxml.com/WPtableGrid.php

I found this works for me best:

```js
  const tbl = new Table({
    rows,
    width: {
      size: 5000,
      type: WidthType.PERCENTAGE,
    },
  });
  const grid = findXMLComponent(tbl, 'w:tblGrid');
  if (grid) {
    tbl.root.splice(tbl.root.indexOf(grid), 1);
  }
```

**Comment by dolanmiu** [comment]
Yup, it's set to 100 by default on purpose (I randomly picked 100 at the time of writing the code)

So this is a non-issue I suppose?

**Comment by tripodsan** [comment]
> So this is a non-issue I suppose?

yes, it is an issue. if you omit the `columnWidths` when creating the table, you add widths with `100`. this makes the tables unusable on online-word. the correct behaviour would be to **not** create a `w:tblGrid` component if no `columnWidths` is specified. this makes word to auto size the columns.

**Comment by dolanmiu** [comment]
Makes sense, will make a fix
